iam/iam-policy-for-ecs/core/main.tf

```diff
@@ -0,0 +1,103 @@
+data "aws_iam_user" "main" {
+  user_name = var.username
+}
+
+locals {
+  staging_developer_access_projects    = try(data.aws_iam_user.main.tags.staging-developer-access-projects, "")
```

Comment:
is it possible to keep this general, i.e. not bound to environments named `staging` and `production`?

Reply:
Can do if we pass the projects and environment as variables.
The tradeoff is we can't automatically assign the permission based on the user tags using the modules
(We have to extract the values and passing them as variables into the modules)
Both have its pros and cons
Option 1: Extracting the tag values inside module -> Simplified the usage of module, i.e. just need to pass the username into the module, less flexibility
 Option 2: Making the variable more general -> More flexibility, need to extract the values of projects name and environment from tag (if relevant) and pass into the modules